addons/website/static/src/js/widgets/link_popover_widget.js

```diff
@@ -19,6 +22,22 @@ weWidgets.LinkPopoverWidget.include({
 
         return this._super(...arguments);
     },
+    /**
+     * In edit mode, open website page links in backend mode using the '/@/' controller.
```

Comment:
```suggestion
     * Opens website page links in backend mode by forcing the '/@/' controller.
```
- (Important) Max 80 char
- (Ideally) Start with a verb (with `s` at the end)
- (Nitpicking) We know we are in edit mode, this file is for link preview which is an edit mode feature